[BUG] install_deps on macOS failed

**Describe the bug**
I try to build GraphScope on macOs in standalone mode, and I install dependencies, but it failed when installing zetcd.

**To Reproduce**
./install_deps.sh --dev --cn


**Screenshots**
`[2022-06-01T07:20:56+0800]: Installing zetcd.
go: found github.com/etcd-io/zetcd/cmd/zetcd in github.com/etcd-io/zetcd v0.0.5
go: finding module for package github.com/coreos/zetcd
go: finding module for package golang.org/x/net/context
go: finding module for package github.com/coreos/etcd/pkg/transport
go: finding module for package github.com/coreos/etcd/clientv3
go: finding module for package github.com/coreos/zetcd/zk
go: finding module for package github.com/coreos/zetcd/version
go: downloading github.com/coreos/etcd v0.9.4
go: finding module for package github.com/coreos/zetcd/xchk
go: found github.com/coreos/zetcd in github.com/coreos/zetcd v0.0.5
go: found github.com/coreos/zetcd/version in github.com/coreos/zetcd v0.0.5
go: found github.com/coreos/zetcd/xchk in github.com/coreos/zetcd v0.0.5
go: found github.com/coreos/zetcd/zk in github.com/coreos/zetcd v0.0.5
go: found golang.org/x/net/context in golang.org/x/net v0.0.0-20220531201128-c960675eff93
go: finding module for package github.com/coreos/etcd/pkg/transport
go: finding module for package github.com/coreos/etcd/clientv3
go: finding module for package github.com/coreos/etcd/clientv3/concurrency
go: finding module for package github.com/golang/glog
go: found github.com/golang/glog in github.com/golang/glog v1.0.0
go: finding module for package github.com/coreos/etcd/pkg/transport
go: finding module for package github.com/coreos/etcd/clientv3
go: finding module for package github.com/coreos/etcd/clientv3/concurrency
go: finding module for package github.com/coreos/etcd/pkg/transport
go: finding module for package github.com/coreos/etcd/clientv3
go: finding module for package github.com/coreos/etcd/clientv3/concurrency
go: finding module for package github.com/coreos/etcd/pkg/transport
go: finding module for package github.com/coreos/etcd/clientv3
go: finding module for package github.com/coreos/etcd/clientv3/concurrency
../../../go/pkg/mod/github.com/etcd-io/zetcd@v0.0.5/cmd/zetcd/zetcd.go:27:2: reading github.com/coreos/etcd/go.mod at revision v0.9.4: unknown revision v0.9.4
../../../go/pkg/mod/github.com/coreos/zetcd@v0.0.5/zketcd.go:27:2: reading github.com/coreos/etcd/go.mod at revision v0.9.4: unknown revision v0.9.4
../../../go/pkg/mod/github.com/etcd-io/zetcd@v0.0.5/cmd/zetcd/zetcd.go:28:2: reading github.com/coreos/etcd/go.mod at revision v0.9.4: unknown revision v0.9.4`

**Environment (please complete the following information):**
 - GraphScope version: main
 - OS: MacOS
 - Version 12.3.1
 - Kubernetes Version : none
 - go version: 1.15.14

**Additional context**
I looked up the repository under CoreOs, it doesn't contain etcd, which may cause the error. 

Maybe it's a problem for zetcd, but it seems stop maintenance for a long time.

--- a/scripts/install_deps.sh
+++ b/scripts/install_deps.sh
@@ -185,6 +185,7 @@ init_basic_packages() {
       git
       rapidjson-dev
       libmsgpack-dev
+      golang-go
     )
   elif [[ "${PLATFORM}" == *"CentOS"* ]]; then
     BASIC_PACKGES_TO_INSTALL=(
@@ -223,6 +224,7 @@ init_basic_packages() {
       curl
       rapidjson-devel
       msgpack-devel
+      golang
     )
   else
     BASIC_PACKGES_TO_INSTALL=(
@@ -242,6 +244,7 @@ init_basic_packages() {
       libomp
       rapidjson
       msgpack-cxx
+      go
     )
   fi
   readonly BASIC_PACKGES_TO_INSTALL
@@ -334,16 +337,6 @@ check_dependencies() {
      ( ! command -v ${HOME}/.cargo/bin/rustup &> /dev/null || \
     [[ "$(${HOME}/.cargo/bin/rustc --V | awk -F ' ' '{print $2}')" < "1.52.0" ]] ); then
     packages_to_install+=(rust)
-  fi
-
-  # check go < 1.16 (reason: vertion 1.16 can't install zetcd)
-  if $(! command -v go &> /dev/null) || \
-     [[ "$(go version 2>&1 | awk -F '.' '{print $2}' | awk -F ' ' '{print $1}')" -ge "16" ]]; then
-    if [[ "${PLATFORM}" == *"CentOS"* ]]; then
-      packages_to_install+=(golang)
-    else
-      packages_to_install+=(go)
-    fi
   fi
 
   # check etcd
@@ -555,16 +548,6 @@ install_dependencies() {
     log "Installing packages ${BASIC_PACKGES_TO_INSTALL[*]}"
     sudo apt-get install -y ${BASIC_PACKGES_TO_INSTALL[*]}
 
-    if [[ "${packages_to_install[*]}" =~ "go" ]]; then
-      # packages_to_install contains go
-      log "Installing Go."
-      wget -c https://golang.org/dl/go1.15.5.linux-amd64.tar.gz -P /tmp
-      sudo tar -C /usr/local -xzf /tmp/go1.15.5.linux-amd64.tar.gz
-      rm -fr /tmp/go1.15.5.linux-amd64.tar.gz
-      sudo ln -sf /usr/local/go/bin/go /usr/local/bin/go
-      # remove go from packages_to_install
-      packages_to_install=("${packages_to_install[@]/go}")
-    fi
     if [[ "${packages_to_install[*]}" =~ "rust" ]]; then
       # packages_to_install contains rust
       log "Installing rust."
@@ -582,15 +565,6 @@ install_dependencies() {
       sudo apt install -y libarrow-dev=3.0.0-1 libarrow-python-dev=3.0.0-1
       # remove apache-arrow from packages_to_install
       packages_to_install=("${packages_to_install[@]/apache-arrow}")
-    fi
-
-    if [[ "${packages_to_install[*]}" =~ "zetcd" ]]; then
-      log "Installing zetcd."
-      export PATH=${PATH}:/usr/local/go/bin
-      go get github.com/etcd-io/zetcd/cmd/zetcd
-      sudo cp ${HOME}/go/bin/zetcd /usr/local/bin/zetcd
-      # remove zetcd from packages_to_install
-      packages_to_install=("${packages_to_install[@]/zetcd}")
     fi
 
     if [[ ! -z "${packages_to_install}" ]]; then
@@ -629,15 +603,6 @@ install_dependencies() {
       popd
       rm -fr /tmp/openmpi-4.0.5 /tmp/openmpi-4.0.5.tar.gz
       packages_to_install=("${packages_to_install[@]/openmpi}")
-    fi
-
-    if [[ "${packages_to_install[*]}" =~ "zetcd" ]]; then
-      log "Installing zetcd."
-      export PATH=${PATH}:/usr/local/go/bin
-      go get github.com/etcd-io/zetcd/cmd/zetcd
-      sudo cp ${HOME}/go/bin/zetcd /usr/local/bin/zetcd
-      # remove zetcd from packages_to_install
-      packages_to_install=("${packages_to_install[@]/zetcd}")
     fi
 
     if [[ "${packages_to_install[*]}" =~ "etcd" ]]; then
@@ -730,26 +695,6 @@ install_dependencies() {
       packages_to_install=("${packages_to_install[@]/openjdk@11}")
     fi
 
-    if [[ "${packages_to_install[*]}" =~ "go" ]]; then
-      # packages_to_install contains go
-      log "Installing Go."
-      wget -c https://dl.google.com/go/go1.15.15.darwin-amd64.pkg -P /tmp
-      sudo installer -pkg /tmp/go1.15.15.darwin-amd64.pkg -target /
-      rm -fr /tmp/go1.15.15.darwin-amd64.pkg
-      sudo ln -sf /usr/local/go/bin/go /usr/local/bin/go
-      # remove go from packages_to_install
-      packages_to_install=("${packages_to_install[@]/go}")
-    fi
-
-    if [[ "${packages_to_install[*]}" =~ "zetcd" ]]; then
-      log "Installing zetcd."
-      export PATH=/usr/local/go/bin:${PATH}
-      go get github.com/etcd-io/zetcd/cmd/zetcd
-      sudo cp ${HOME}/go/bin/zetcd /usr/local/bin/zetcd
-      # remove zetcd from packages_to_install
-      packages_to_install=("${packages_to_install[@]/zetcd}")
-    fi
-
     if [[ "${packages_to_install[*]}" =~ "rust" ]]; then
       # packages_to_install contains rust
       log "Installing rust."
@@ -764,13 +709,10 @@ install_dependencies() {
       packages_to_install=("${packages_to_install[@]/maven}")
     fi
 
-
     if [[ ! -z "${packages_to_install}" ]]; then
       log "Installing packages ${packages_to_install[*]}"
       brew install ${packages_to_install[*]}
     fi
-
-
 
     if [[ "$(uname -m)" == "x86_64" ]]; then
       declare -r homebrew_prefix="/usr/local"
@@ -783,6 +725,14 @@ install_dependencies() {
     export CC=${homebrew_prefix}/opt/llvm/bin/clang
     export CXX=${homebrew_prefix}/opt/llvm/bin/clang++
     export CPPFLAGS=-I${homebrew_prefix}/opt/llvm/include
+  fi
+
+  if [[ "${packages_to_install[*]}" =~ "zetcd" ]]; then
+    log "Installing zetcd."
+    GO111MODULE="auto" go get github.com/etcd-io/zetcd/cmd/zetcd
+    sudo cp ${HOME}/go/bin/zetcd /usr/local/bin/zetcd
+    # remove zetcd from packages_to_install
+    packages_to_install=("${packages_to_install[@]/zetcd}")
   fi
 
   log "Installing python packages for vineyard codegen."

